### PR TITLE
[feat] add action (charge_action_point)

### PR DESCRIPTION
**Pull request by kanade012**
새로운 액션을 추가했습니다.

### CODE REVIEW

**Review by moreal — APPROVED** [review]
LGTM!

File: @planetarium/lib9c/package.json [hunk 1]
```diff
@@ -34,5 +34,6 @@
     "buffer": "^6.0.3",
     "decimal.js": "^10.4.3",
     "uuid": "^9.0.1"
-  }
+  },
+  "packageManager": "pnpm@9.12.2+sha512.22721b3a11f81661ae1ec68ce1a7b879425a1ca5b991c975b074ac220b187ce56c708fe5db69f4c962c989452eee76c82877f4ee80f474cebd61ee13461b6228"
```
Comment (Atralupus) [inline @planetarium/lib9c/package.json:38]: @moreal is this OK?

Comment (moreal) [reply @planetarium/lib9c/package.json]: Hmm, it's not unusual to see packageManager set, but I've never seen it attached to a hash commit.

File: @planetarium/lib9c/package.json [hunk 2]
```diff
@@ -35,4 +35,4 @@
     "decimal.js": "^10.4.3",
     "uuid": "^9.0.1"
   }
-}
+}
```
Comment (Atralupus) [inline @planetarium/lib9c/package.json:38]: ! @kanade012

**Pull request merged**